cms/templates/content_libraries/xblock_iframe.html

```diff
@@ -125,7 +125,7 @@
   <!-- fragment head -->
   {{ fragment.head_html | safe }}
 </head>
-<body>
+<body style="min-height: auto">
```

Comment:
Wow, thanks for researching and finding out about this.

Nit: This is so specific that worth a comment in my opinion.